### ISSUE TITLE
Missing word - Applying SC 3.3.8 

### ISSUE THREAD

**Issue by mr45144** [w3c/wcag2ict#407]
The draft reads:
"This applies directly as written, and as described in [Intent from Understanding Success Criterion 3.3.8](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html), “the Web site” with “a Web site, non-web document, or software”.

Insert "replacing" before "the Website"

**Comment by maryjom** [comment]
Thank you for your comment. This issue has been fixed in the latest WCAG2ICT editor's draft. See [Applying SC 3.3.8 Accessible Authentication (Minimum) to Non-Web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-3-3-8-accessible-authentication-minimum-to-non-web-documents-and-software).
